spec/furniture/marketplace/stripe_events_controller_request_spec.rb

```diff
@@ -3,9 +3,13 @@
 # rubocop:disable RSpec/VerifiedDoubles
 RSpec.describe Marketplace::StripeEventsController, type: :request do
   let(:marketplace) { create(:marketplace, :with_stripe_utility, stripe_account: "sa_1234", stripe_webhook_endpoint_secret: "whsec_1234") }
+  let(:marketplace_with_square) { create(:marketplace, :with_stripe_utility, stripe_account: "sa_1234", stripe_webhook_endpoint_secret: "whsec_1234", square_location_id: "123456789", square_environment: "sandbox", secrets: {square_access_token: "fake_square_access_token"}) }
   let(:space) { marketplace.space }
   let(:member) { create(:membership, space: space).member }
   let(:order) { create(:marketplace_order, :with_products, status: :pre_checkout, marketplace: marketplace) }
+  let(:person) { create(:person) }
+  let(:shopper) { create(:marketplace_shopper, person: person) }
```

Comment:
Why do we need a `Shopper` with a `Person`?

Reply:
Because it's in the data: https://github.com/zinc-collective/convene/blob/adds-square-notifications-tests/app/furniture/marketplace/order.rb#L136.

Reply:
Ahhh, thank you that makes sense. What happens if the `Shopper` does not have a `Person`? `Guest` users for example do not have a `Person` record.

Reply:
@rosschapman - I am sitll a bit worried about what happens when a Guest checks out at a Marketplace with Square. I know it's not the most important thing to deal with now-right-now; since no one has Square set up yet; but a test for that use case would put my mind at ease.

Reply:
@zspencer sure thing. I've got some updates coming soon so I'll add this. Thanks for reminding me about the _Guest_ use case.